src/app/(main)/tickets/[id]/TicketRightContainer.tsx

```diff
@@ -31,10 +33,12 @@ export default async function TicketRightContainer({
 
   return (
     <section className='w-1/5 m-5 p-8 bg-white shadow-sm'>
+      //todo status! to nie jest dobra praktyka, po to masz ts zeby wlasnie edgecase z null ogarnac sobie , możesz też dać np ticktet.status || ""
       <p>{capitalizeFirstLetter(ticket.status!)}</p>
       <hr className='my-3' />
       <p className='text-xs'>PROPERTIES</p>
       <UpdateTicketForm id={id} ticket={ticket} users={users} />
+      //todo to samo co wyzej, z tym ze tu nie dasz || :D zaifuj oba przed returnem
```

Comment:
A mógłbym to zapisać w taki sposób?
      {data?.session && data.session.user.email === ticket.user_email && (
        <DeleteButton id={ticket.id} />
      )}